src/js/tech/html5.js

```diff
@@ -368,24 +368,26 @@ class Html5 extends Tech {
     // playback has started, which triggers the live display erroneously.
     // Return NaN if playback has not started and trigger a durationupdate once
     // the duration can be reliably known.
-    if (this.el_.duration === Infinity &&
```

Comment:
Not strictly perf-related, but I noticed it in searching for `timeupdate` listeners.